packages/monaco/src/editor-hooks.ts

```diff
@@ -8,11 +8,11 @@ class EditorEvents extends EventTarget {
 
 const event = new EditorEvents();
 type EventNames = 'resetCode';
-export function useResetEditor(deps: unknown[] = []): {
-  subscribe: (eventName: EventNames, cb: () => void) => void;
+export function useResetEditor(): {
+  subscribe: (eventName: EventNames, cb: () => void, deps: unknown[]) => void;
   dispatch: (eventName: EventNames) => void;
 } {
-  function subscribe(eventName: EventNames, cb: () => void) {
+  function subscribe(eventName: EventNames, cb: () => void, deps: unknown[] = []) {
```

Comment:
Happy to undo this change but I think it makes more sense to have the dependencies scoped to `subscribe` since that's where they are used.

Reply:
nope this is better. i def just rushed my solution in a couple mins lulz